js/plates-model/rock-properties.ts

```diff
@@ -25,6 +26,12 @@ export interface IRockProperties {
 }
 
 export const ROCK_PROPERTIES: Record<Rock, IRockProperties> = {
+  [Rock.ContinentalSediment]: {
+    label: "Continental Sediments",
+    orderIndex: 0,
+    isTransferrableDuringCollision: true,
+    canSubduct: true
```

Comment:
PT story says "The continental sediment will react to changing P/T conditions as any other rock would, treat it like oceanic sediments."

Reply:
I'd still set canSubduct to false. This property decides whether crust/field can subduct or not. And we assume that only oceanic fields can subduct. That's why only oceanic rocks have this property set to true.

In other words, you could think that canSubduct is a property that will decide whether a crust is oceanic or continental. The name is a bit more generic tho, as we also have islands, etc. Continental sediment is part of a continental crust.